### ISSUE TITLE
Stripe mock data not compatible with djstripe/stripe. 

### ISSUE THREAD

**Issue by jsutaria** [stripe/stripe-mock#447]
Running into the following error when syncing customers from djstripe - 

```
import stripe
from djstripe import models as djstripe_models

stripe_customer = stripe.Customer.create(
    email=user.email, name=user.full_name
)
customer = djstripe_models.Customer.sync_from_stripe_data(stripe_customer)
```

Leads to - 
```
ValueError: You cannot set default_payment_method to an empty string on this object. The empty string is treated specially in our requests. If you'd like to delete the property using the save() method on this object, you may set {
  "address": null,
  "balance": 0,
  "created": 1234567890,
  "currency": "usd",
  "default_source": "",
  "delinquent": false,
  "description": null,
  "discount": null,
  "email": "dcfem++3199@example.com",
  "id": "cus_LG6CoeoStCxpP4E",
  "invoice_prefix": "809458F",
  "invoice_settings": {
    "custom_fields": null,
    "default_payment_method": "",
    "footer": null,
    "rendering_options": null
  },
  "livemode": false,
  "metadata": {},
  "name": "Katherine Anderson",
  "next_invoice_sequence": 1,
  "object": "customer",
  "phone": null,
  "preferred_locales": [],
  "shipping": null,
  "tax_exempt": "none",
  "test_clock": ""
}.default_payment_method = None. Alternatively, you can pass default_payment_method='' to delete the property when using a resource method such as modify().
```

Pinning stripe-mock to v1.170.0 fixes the issue, implying an issue with https://github.com/stripe/stripe-mock/releases/tag/v0.171.0

**Comment by jsutaria** [comment]
CC @anniel-stripe - suspect commit: https://github.com/stripe/stripe-mock/commit/1d80bcc48dea3aab3e43c58bad2dcb455b880433

**Comment by anniel-stripe** [comment]
Hi @jsutaria , thanks for raising this issue! The fix has been released in [v0.172.0](https://github.com/stripe/stripe-mock/releases/tag/v0.172.0).
